gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineClasspathConflict.groovy

```diff
@@ -0,0 +1,62 @@
+/*
+ * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.plugins
+
+import java.util.jar.JarEntry
+import java.util.jar.JarFile
+import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.internal.impldep.com.google.common.collect.HashMultimap
+import org.gradle.internal.impldep.com.google.common.collect.SetMultimap
+
+/**
+ * Adds a simple test for shadowed classes.
+ */
+class BaselineClasspathConflict extends AbstractBaselinePlugin  {
+
+    @Override
+    void apply(Project project) {
+        if (project.plugins.hasPlugin(JavaPlugin)) {
+            def silentConflict = project.tasks.create("silentConflict") {
+                dependsOn: project.configurations.testRuntime
```

Comment:
testRuntime?

Reply:
think we should set the default, but let plugin users specify the configuration (possibly multiple)